user/articles/06-app-management/07-disaster-recovery.markdown

```diff
@@ -0,0 +1,155 @@
+# Disaster Recovery
+
+
+
+## Introduction
+
+DXP Cloud provides two ways for customers to take advantage of the Disaster Recovery (DR) procedure in the case of major incidents.
+
+- **Automatic Disaster Recovery:** It's important to understand that <u>DXP Cloud already performs automatic disaster recovery by replicating services between three Availability Zones in different geographic locations within the same Region</u>. In case any of the Availability Zones become unavailable, the Load Balancer automatically routes to the remaining Availability Zone without requiring DNS changes on the customer side. In this situation, no action is required from the customer during an incident.
```

Comment:
@zenorocha let's change the name from automatic to "Intra Region"?

Reply:
It's a good suggestion to rename it to "Intra Region", although it is also important to make it clear this is automatic or auto-of-the-box functionality without any customer action.